pmm/plugin/mysql/mysql_test.go

```diff
@@ -223,34 +223,17 @@ func TestGetMysqlInfo(t *testing.T) {
 
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
 	defer cancel()
-	res := *getMysqlInfo(ctx, db)
-	expected := MySQLInfo{
+	info := *getInfo(ctx, db)
```

Comment:
2-valued getInfo(ctx, db) (value of type (*github.com/percona/pmm-client/pmm/plugin.Info, error)) where single value is expected